lib/android/app/src/test/java/com/reactnativenavigation/viewcontrollers/modal/ModalStackTest.java

```diff
@@ -173,6 +173,13 @@ public void onSuccess(String childId) {
         verifyZeroInteractions(listener);
     }
 
+    @Test
+    public void dismissAllModals_rejectIfEmpty() {
```

Comment:
Bit of a nit pick but maybe change to resolveIfEmpty since you no longer reject

Reply:
Good point 😅